applications/nrf_desktop/src/modules/hid_state_pm.c

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/types.h>
+
+#include "hid_event.h"
+
+#define MODULE hid_state_pm
+#include <caf/events/module_state_event.h>
+#include <caf/events/keep_alive_event.h>
+
+
+static bool event_handler(const struct event_header *eh)
+{
+	__ASSERT_NO_MSG(is_hid_report_event(eh));
```

Comment:
This adds additional event handling to our critical path. We may need to verify if this has a negative impact on report rate.

Reply:
I could make a check in release configuration